specification/cognitiveservices/data-plane/FormRecognizerReceipt/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -0,0 +1,261 @@
+    "hostTemplate": "{Endpoint}/formrecognizer/v1.0-preview",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "Ocr.json#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+
+  "paths": {
+    "/prebuilt/receipt/asyncBatchAnalyze": {
+      "post": {
+        "description": "Batch Read Receipt operation. The response contains a field called 'Operation-Location', which contains the URL that you must use for your 'Get Read Receipt Result' operation.",
+        "operationId": "BatchReadReceipt",
+        "parameters": [
+          {
+            "$ref": "Ocr.json#/parameters/ImageUrl"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header. The 'Operation-Location' URL will expire in 48 hours.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation. The URL will expire in 48 hours. ",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "Ocr.json#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful 'Batch Read Receipt' request": {
+            "$ref": "./examples/SuccessfulBatchReadReceiptWithUrl.json"
+          }
+        }
+      }
+    },
+
+    "/prebuilt/receipt/operations/{operationId}": {
+      "get": {
+        "description": "This interface is used for getting the analysis results of a 'Batch Read Receipt' operation. The URL to this interface should be retrieved from the 'Operation-Location' field returned from the 'Batch Read Receipt' operation.",
+        "operationId": "GetReadReceiptResult",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Id of read operation returned in the response of a 'Batch Read Receipt' operation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the read operation status.",
+            "schema": {
+              "$ref": "#/definitions/readReceiptResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "Ocr.json#/definitions/ComputerVisionError"
+            }
+          }
+        }
```

Comment:
Need to add:
        "x-ms-examples": {
          "Successful 'Get Read Receipt Result' request": {
            "$ref": "./examples/SuccessfulReadReceiptResult.json"
          }
        }

Reply:
done.